internal/net/net.go

```diff
@@ -36,30 +38,38 @@ const (
 	defaultPort = 80
 )
 
-type Conn = net.Conn
-type Dialer = net.Dialer
-type ListenConfig = net.ListenConfig
-type Listener = net.Listener
-type Resolver = net.Resolver
+type (
+	Conn         = net.Conn
+	Dialer       = net.Dialer
+	ListenConfig = net.ListenConfig
+	Listener     = net.Listener
+	Resolver     = net.Resolver
```

Comment:
```suggestion
	// Conn is a wrapper of net.Conn
	Conn         = net.Conn
	
	// Dialer is a wrapper of net.Dialer
	Dialer       = net.Dialer
	
	// ListenConfig is a wrapper of net.ListenConfig
	ListenConfig = net.ListenConfig
	
	// Listener is a wrapper of net.Listener
	Listener     = net.Listener
	
	// Resolver is a wrapper of net.Resolver
	Resolver     = net.Resolver
```

Reply:
maybe the word `alias` is suitable than `wrapper`